server/src/routes/upload/parseBeatmap.ts

```diff
@@ -101,6 +101,26 @@ export const parseBeatmap: (
     hash.update(b)
   }
 
+  const difficultiesData = await Promise.all(
+    [1, 3, 5, 7, 9].map(async rank => {
+      const diff = difficulties.find(x => x._difficultyRank === rank)
+      if (!diff) {
+        return null
+      }
+      try {
+        const diffContent = await zip.file(diff._beatmapFilename).async('text')
+        const diffData: IDifficultyJSON = JSON.parse(diffContent)
+        return {
+          duration: Math.max(...diffData._notes.map(note => note._time)),
+          notes: diffData._notes.length,
```

Comment:
This will count both notes and bombs as notes.
A separate key for bombs would be nice.

Bombs have a `_type` of `3`